tests/lib/prepare-project.sh

```diff
@@ -7,8 +7,21 @@ create_test_user(){
         # the all-snap, which has its own user & group database.
         # Nothing special about 12345 beyond it being high enough it's
         # unlikely to ever clash with anything, and easy to remember.
-        addgroup --quiet --gid 12345 test
-        adduser --quiet --uid 12345 --gid 12345 --disabled-password --gecos '' test
```

Comment:
That still doesn't help that the `useradd` tool doesn't support `--disable-password` or `--gecos`. However reworked the code a bit to look better and have more in common across different distros.